internal/internal_task_handlers_test.go

```diff
@@ -1279,23 +1279,26 @@ func (t *TaskHandlersTestSuite) TestHeartBeat_Interleaved() {
 
 	heartbeatErr := cadenceInvoker.BatchHeartbeat([]byte("1"))
 	t.NoError(heartbeatErr)
-	time.Sleep(1 * time.Second)
+	time.Sleep(1000 * time.Millisecond)
 
 	for i := 0; i < 4; i++ {
 		heartbeatErr = cadenceInvoker.BackgroundHeartbeat()
 		t.NoError(heartbeatErr)
-		time.Sleep(time.Second)
+		time.Sleep(800 * time.Millisecond)
 	}
 
+	time.Sleep(time.Second)
+
 	heartbeatErr = cadenceInvoker.BatchHeartbeat([]byte("2"))
 	t.NoError(heartbeatErr)
-	time.Sleep(1 * time.Second)
+	time.Sleep(1000 * time.Millisecond)
 
 	for i := 0; i < 4; i++ {
 		heartbeatErr = cadenceInvoker.BackgroundHeartbeat()
 		t.NoError(heartbeatErr)
-		time.Sleep(time.Second)
+		time.Sleep(800 * time.Millisecond)
```

Comment:
Why is this different?

Reply:
The test is expecting 3 calls (with 1s) within 3 sec. This is causing flakiness. So I reduced the interval to fix the flakiness.